src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -1843,6 +1843,35 @@ export function pickClinicalAttrColorsByIndex(
     );
 }
 
+export function pickClinicalAttrColorByIndex(
```

Comment:
thanks for making this change! I think the name is too detailed for users, maybe just `pickUnusedColor`